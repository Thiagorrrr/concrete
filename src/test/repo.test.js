import React from 'react';
import { render } from '@testing-library/react';
import Repo from '../components/molecules/repo';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<Repo />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});