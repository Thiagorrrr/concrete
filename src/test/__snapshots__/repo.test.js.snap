// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="repo"
>
  <ul
    className="repo__list"
  >
    <div
      className="notFound"
    >
      <span
        className="NotFound"
      >
          NotFound
      </span>
    </div>
  </ul>
</div>
`;
